_ingest-pipelines/processors/remove.md

```diff
@@ -30,7 +30,7 @@ The following table lists the required and optional parameters for the `remove`
 
 | Parameter  | Required/Optional  | Description  |
 |---|---|---|
-`field`  | Required  | The name of the field containing the data to be appended. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). |
+`field`  | Required  | The name of the field containing the data to be appended. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). Note that these metadata fields `_index`, `_version` and `_version_type` are not allowed to remove, and `_id` is also not allowed to remove when there's a specified external version for the ingesting document. |
```

Comment:
```suggestion
`field`  | Required  | The name of the field containing the data to be appended. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). The following metadata fields are not allowed to be removed: `_index`, `_version`, `_version_type`, and `_id`. Note that `_id` is not allowed to be removed when there's a specified external version for the ingesting document. |
```

Reply:
Should "removed" be "deleted"

Reply:
Thanks for your review, I think `removed` is better because it's consistent with the name of this processor.

Reply:
Another finding is that `The name of the field containing the data to be appended` should be `... to be removed`, I've changed that, but I see the document of [uppercase processor ](https://github.com/opensearch-project/documentation-website/blob/main/_ingest-pipelines/processors/uppercase.md) also has this problem, we need to open another PR to correct it.

Reply:
@gaobinlong I'll make the update to the uppercase processor and send to you for tech review approval. Thank you :)

Reply:
@gaobinlong Change made to uppercase.md. https://github.com/opensearch-project/documentation-website/pull/5925/files